core/types/bundle_gasless.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+type SimulateGaslessBundleArgs struct {
```

Comment:
Do we have to call it as `Gasless`? because it can  also be used to simulate for common bundle.

Reply:
I think not, common bundle has gas floor, and has reverting hashes param. their logic are different.